lib/ansible/modules/cloud/amazon/ecs_tag.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Michael Pechner <mikey@mikey.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: ecs_tag
+short_description: create and remove tags on ecs clusters.
```

Comment:
```suggestion
short_description: create and remove tags on Amazon ECS resources
```

a) short_description shouldn't end in a '.'
b) ECS is an acronym and product name.  In the documentation we should try to refer to it as 'ECS' rather than 'ecs' as an accuracy thing.
c) If I'm reading the module correctly it doesn't just tag clusters...